### ISSUE TITLE
property-no-unknown incorrectly flags scss @use namespace

### ISSUE THREAD

**Issue by johncrim** [kristerkari/stylelint-scss#426]
It appears that the `property-no-unknown` rule is incorrectly flagging SCSS `@use` namespaces, when setting a variable within a namespace. This occurs only within `@mixin`s and `@function`s, it doesn't occur when reading or writing module properties in the global scope. It also only occurs while setting a variable - it doesn't occur when reading the variable.

```scss
@use 'accoutrement/sass' as a;
@use 'sass:list';

// No stylelint error
a.$sizes: ();

@function clear-sizes() {
  // Lint error: Unexpected unknown property "a.$sizes"   property-no-unknown
  a.$sizes: ();
  @return a.$sizes;
}
$s: clear-sizes();

@mixin dump-sizes() {
  // No stylelint error
  $sizes: a.$sizes;
  @debug '# of sizes registered: #{ list.length($sizes) }';
}
@include dump-sizes();
```

The attached file provides a simple repro case with dependencies.
[stylelint-module-property.zip](https://github.com/kristerkari/stylelint-scss/files/4037883/stylelint-module-property.zip)

**Comment by kristerkari** [comment]
Could you please open this issue in https://github.com/stylelint/stylelint which has the mentioned rule.

**Comment by johncrim** [comment]
I opened the issue [in stylelint here](https://github.com/stylelint/stylelint/issues/4544). The reason I originally opened the issue here is that since it's SCSS-specific, it might require SCSS-specific handling.

**Comment by kristerkari** [comment]
Thanks @johncrim ! This repository is not actually related to handling the SCSS syntax for stylelint, but instead it contains additional rules that are meant to be used with Sass.
